tests/unit/database_tests.py

```diff
@@ -1413,32 +1413,14 @@ def test_get_search_result_executes_search_query(self):
         )
         self.assertEqual(len(resp['rows']), 5)
         self.assertTrue(resp['bookmark'])
-        resp.pop('bookmark')
+
         for row in resp['rows']:
             self.assertTrue(row['doc']['_rev'].startswith('1-'))
-            row['doc'].pop('_rev')
+            self.assertEqual('julia', row['fields']['name'])
+
         self.assertEqual(
-            resp,
-            {'rows': [{'fields': {'name': 'julia'}, 'doc': {'_id': 'julia000',
-                                                            'age': 0,
-                                                            'name': 'julia'},
-                       'id': 'julia000', 'order': ['julia000', 0]},
-                      {'fields': {'name': 'julia'}, 'doc': {'_id': 'julia001',
-                                                            'age': 1,
-                                                            'name': 'julia'},
-                       'id': 'julia001', 'order': ['julia001', 0]},
-                      {'fields': {'name': 'julia'},'doc': {'_id': 'julia002',
-                                                           'age': 2,
-                                                           'name': 'julia'},
-                       'id': 'julia002', 'order': ['julia002', 0]},
-                      {'fields': {'name': 'julia'}, 'doc': {'_id': 'julia003',
-                                                            'age': 3,
-                                                            'name': 'julia'},
-                       'id': 'julia003', 'order': ['julia003', 0]},
-                      {'fields': {'name': 'julia'},
-                       'doc': {'_id': 'julia004', 'age': 4,
-                               'name': 'julia'},
-                       'id': 'julia004', 'order': ['julia004', 1]}], 'total_rows': 100}
+            ['julia000', 'julia001', 'julia002', 'julia003', 'julia004'],
+            [row['id'] for row in resp['rows']]
```

Comment:
This is only asserting `_id`. What about asserting the other fields? i.e. `age`, `name`, `fields`, `order`